lib/jekyll/utils.rb

```diff
@@ -146,11 +134,13 @@ def parse_date(input, msg = "Input could not be parsed.")
     # Determines whether a given file has
     #
     # Returns true if the YAML front matter is present.
+    # rubocop: disable PredicateName
```

Comment:
Rubocop suggest to rename `has_yaml_header?` to `yaml_header?` but can we do that?